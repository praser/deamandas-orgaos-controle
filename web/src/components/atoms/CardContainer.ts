import styled from 'styled-components';

const CardContainer = styled.div`
  /* Layout */
  display: flex;
  flex-direction: column;
  min-height: 100px;
  margin-bottom: 24px;
  width: 100%;
  position: relative;

  /* Text */
  font-family: Nunito, -apple-system, system-ui, 'Segoe UI', Roboto,
    'Helvetica Neue', Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji',
    'Segoe UI Symbol', 'Noto Color Emoji';
  font-size: 16px;
  font-weight: 400;
  line-height: 24px;
  text-align: left;
  text-size-adjust: 100%;

  /* Appearance */
  color: rgb(133, 135, 150);
  background-clip: border-box;
  background-color: rgb(255, 255, 255);
  border-color: rgb(227, 230, 240);
  border-style: solid;
  border-width: 1px;
  box-shadow: rgba(58, 59, 69, 0.15) 0px 2.4px 28px 0px;

  /* Other */
  border-radius: 5.6px;
`;

export default CardContainer;
