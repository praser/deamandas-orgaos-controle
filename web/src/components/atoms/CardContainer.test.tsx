import React from 'react';
import { render, screen } from '@testing-library/react';
import CardContainer from './CardContainer';

describe('CardContainer component', () => {
  let cardcontainer: HTMLElement;
  const testId = 'testid';

  beforeEach(() => {
    render(<CardContainer data-testid={testId} />);
    cardcontainer = screen.getByTestId(testId);
  });

  it('is expected to match the snapshot', () => {
    expect(cardcontainer).toMatchSnapshot();
  });
});
