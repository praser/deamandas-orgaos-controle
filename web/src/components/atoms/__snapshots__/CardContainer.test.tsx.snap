// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardContainer component is expected to match the snapshot 1`] = `
<div
  class="sc-bdvvaa fSPymo"
  data-testid="testid"
/>
`;
